tests/geospatial/test_rechunking.py

```diff
@@ -1,25 +1,39 @@
-import pytest
 import xarray as xr
 from coiled.credentials.google import CoiledShippedCredentials
 
 
-@pytest.mark.client("era5_rechunking")
-def test_era5_rechunking(client, gcs_url, scale):
-    # Load dataset
-    ds = xr.open_zarr(
-        "gs://weatherbench2/datasets/era5/1959-2023_01_10-full_37-1h-0p25deg-chunk-1.zarr",
-    ).drop_encoding()
+def test_era5_rechunking(
+    gcs_url,
+    scale,
+    client_factory,
+    cluster_kwargs={
+        "workspace": "dask-engineering-gcp",
+        "region": "us-central1",
+        "wait_for_workers": True,
+    },
+    scale_n_workers={
+        "small": 10,
+        "large": 100,
+    },
```

Comment:
This change came out of needing to now manage having different configs for "small"/"large" versions of tests that run on multiple cloud providers. I thought inlining the spec for what "small"/"large" mean and other config (mostly related to the dataset being used) like "region"/"workspace" directly in the test definition felt nice, and was pretty easy to get working, but am open to other options.

Reply:
Noting that @hendrikmakait and I chatted about this offline earlier today. We both agreed that this is a fine start and we can always refactor as we start adding more workflows and have a better picture of what needs we'll have. So go with this approach for now and change things later as needed.